### ISSUE TITLE
x/pkgsite: markdown is broken in my readme.md

### ISSUE THREAD

**Issue by rogercarter1** [golang/go#40607]
### What is the URL of the page with the issue?
 https://pkg.go.dev/mod/github.com/tachyon-protocol/goVsysSdk@v0.1.0
 
 
### What is your user agent?
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4222.0 Safari/537.36
 
 
 
### Screenshot
<img width="1039" alt="Screen Shot 2020-08-06 at 3 21 25 PM" src="https://user-images.githubusercontent.com/53250825/89503017-85826d00-d7f8-11ea-8437-537b379ea322.png">

 
 
 
### What did you do?
* write golang example code with  ``` in my readme.md 
 
### What did you expect to see?
 * markdown should be view as good as https://github.com/tachyon-protocol/goVsysSdk/blob/v0.1.0/README.md
 
 
### What did you see instead?
* markdown is broken

**Comment by shaqque** [comment]
This is related to #39297. In particular, the README in https://pkg.go.dev/mod/github.com/tachyon-protocol/goVsysSdk@v0.1.0 doesn't have a space/newline before the fenced code block, causing blackfriday to not detect this as a code block.

Using the same code I posted in the linked issue above with minor tweaks, I managed to confirm that this is indeed the case, and that adding a newline before the fenced code block will allow blackfriday to correctly detect and render the code block. I also tried using goldmark on the readme _without_ adding the space/newline before the code block, and goldmark still correctly identified and rendered the code block.
